dark/assets/css/styles.css

```
@@ -29,7 +26,12 @@
   color: black;
   background: var(--primary-color);
 }
-
+/*  Utility */

```

Add a new line before this @priyankarpal 
```suggestion

/*  Utility */
```

---

Don't resolve the conversation without doing the changes @priyankarpal 
Do these changes first

---

check I deleted my own branch. I change HTML and CSS code to main.